simulation_ws/src/object_tracker_simulation/package.xml

```diff
@@ -9,16 +9,22 @@
   <author email="ros-contributions@amazon.com">AWS RoboMaker</author>
   <maintainer email="ros-contributions@amazon.com">AWS RoboMaker</maintainer>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament</buildtool_depend>
```

Comment:
There's no ament, maybe ament_package?

Reply:
Agree. There's no ament. hence we will remove it on commit d1a8bea

Reply:
I do think you need to depend on some package for buildtool. I think it's supposed to be `ament_package` for Python ament packages.

Reply:
Since It is pure python package, so we will use "ament_package" package as buildtool for Python packages.

[ament_package](https://github.com/ament/ament_package) is a Python package providing an API to parse the files containing the meta information.

[commit] https://github.com/aws-robotics/aws-robomaker-sample-application-objecttracker/pull/31/commits/2fcee0945e94e56c3794a61b34afb7e8e9423030